docs/shadow_config_spec.md

```diff
@@ -293,7 +293,7 @@ The queueing discipline to use at the network interface.
 
 #### `experimental.interpose_method`
 
-Default: "ptrace"  
+Default: "preload"
```

Comment:
Needs the two spaces for the line break.

Reply:
Ohhhh right. Thanks!